tempesta_fw/t/unit/test_tfw_str.c

```diff
@@ -282,20 +311,20 @@ TEST(tfw_str_eq_cstr, handles_empty_strs)
 {
 	TfwStr s1 = {
 		.len = 0,
-		.ptr = (void *)"garbage"
+		.data = (char *)"garbage"
```

Comment:
`{ .data = "xxxxx" }` is sufficient here. All other members will be initialized to zero values.